lib/ansible/modules/clustering/kubevirt/k8s_service.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# Apache License, Version 2.0
+# (see LICENSE or http://www.apache.org/licenses/LICENSE-2.0)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: k8s_service
+
+short_description: Manage Services on Kubernetes
+
+description:
+    - Use Openshift Python SDK to manage Services on Kubernetes
+
+version_added: "2.8"
+
+author: KubeVirt Team (@kubevirt)
+
+options:
+    name:
+        description:
+            - Use to specify a Service object name.
+        required: true
+        type: str
+    namespace:
+        description:
+            - Use to specify a Service object namespace.
+        required: true
+        type: str
+    type:
+        description:
+            - Specifies the type of Service to create.
+            - https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
```

Comment:
you can use U(https://url) tag, to make it real link

Reply:
Also I would be here more precise, this is what k8s doc says:
```
type determines how the Service is exposed. Defaults to ClusterIP. Valid options are ExternalName, ClusterIP, NodePort, and LoadBalancer. "ExternalName" maps to the specified externalName. "ClusterIP" allocates a cluster-internal IP address for load-balancing to endpoints. Endpoints are determined by the selector or if that is not specified, by manual construction of an Endpoints object. If clusterIP is "None", no virtual IP is allocated and the endpoints are published as a set of endpoints rather than a stable IP. "NodePort" builds on ClusterIP and allocates a port on every node which routes to the clusterIP. "LoadBalancer" builds on NodePort and creates an external load-balancer (if supported in the current cloud) which routes to the clusterIP. More info: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services---service-types
```
Maybe at least add link to that doc?